CHANGELOG.md

```diff
@@ -2,86 +2,63 @@
-* `use_item`, (@BastienDuplessier)
-* `test_item`, (@BastienDuplessier)
-
-### Corrections de bogues
-
-* Correction de `me` dans les trajectoire d'événement (@xvw, reporté par @Zer0Zer0x)
-* Correction des commandes pour flasher les cases (@xvw)
-* Correction de la commande `picture_zoom` (@xvw)
-* Correction de la commande `event_priority` (@xvw, reporté par @Zer0Zer0x)
-* Correction de la commande `event_trigger` (@xvw)
+### additions
+* Add range support in the syntax shortcuts for variables : `SV[1, 2, 1..18] = 10` (for example) (@xvw)
+* Add command `camera_scrolling?` (@xvw, reported by @Zer0Zer0x)
+* Add commandsn `windowskin_tone` and `windowskin_opacity` (@xvw, reported by @YendaHusk)
+* Add commands `event_tone` and `player_tone` (@xvw, reported by @Zer0Zer0x)
+* Add command `event_character_name` (@BastienDuplessier)
+* Add command `event_character_index` (@BastienDuplessier)
+* Add command `event_change_character` (@BastienDuplessier)
+*    - Add command `menu_enabled?|disabled?` 
+     - Add command `save_enabled?|disabled?`
+     - Add command `formation_enabled?|disabled?`
+     - Add command `encounter_enabled?|disabled?`
+     (@xvw, reported by Zer0Zer0x)
+* Move command `current_event_id` and `me` from EvEx to Commands (@BastienDuplessier)
+* Add command `use_item` (@BastienDuplessier)
+* Add command `test_item` (@BastienDuplessier)
+
+### Bug fixes
+* Fix `me` on event's trips (@xvw, reported by @Zer0Zer0x)
+* Fix `flash_*` commands (@xvw)
+* Fix command `picture_zoom` (@xvw)
+* Fix command `event_priority` (@xvw, reported by @Zer0Zer0x)
+* Fix command `event_trigger` (@xvw)
 
 ## Release (1.1.0)
 
-### Commandes ajoutées
-
-*  `last_used_item`, (@hinola, reporté par @Zer0Zer0x)
-*  `disable_weather_dimness`+ `disable_weather_dimness` (@BastienDuplessier, reporté par @Zer0Zer0x)
-*  `window_exists?` (@hinola)
-*    - `camera_locked?` 
-     - `camera_lock_x`
-     - `camera_lock_y`
-     - `camera_unlock_x`
-     - `camera_unlock_y` 
-     (@BastienDuplessier, reporté par @lunri)
-*  `mouse_hover_window?` (@hinola)
-*  `choice` + `last_choice` (@xvw reporté par @Zer0Zer0x)
-*  - `dash_activation`
-   - `dash_activate`
-   - `dash_deactivate`
-   - `dash_activate?`
-   - `dash_deactivate?`
-   (@xvw, reporté par @Zer0Zer0x)
-
-### Corrections de bogues
-*  Correction des commandes `armors_possessed` et `weapons_possessed` (@hinola)
-*  Corrections des commandes `event_flash` et `player_flash` (@BastienDuplessier)
-*  Correction de l'afficheur de texte (profil avec contours) (@xvw reporté par @jauke)
-*  Correction des interpolations linéaires (@xvw reporté par @hinola)
-
+### additions
+*  Add command `last_used_item`, (@hinola, reported by @Zer0Zer0x)
+*  Add command `disable_weather_dimness`+ `disable_weather_dimness` (@BastienDuplessier, reported by @Zer0Zer0x)
+*  Add command `window_exists?` (@hinola)
+*    - Add command `camera_locked?` 
```

Comment:
The leading bullet's symbol (`*`) should be removed. Could you remove the **trailing whitespace** as well ? [#sameplayershootagaint]